vello_tests/tests/known_issues.rs

```diff
@@ -82,15 +82,36 @@ const DATA_IMAGE_PNG: &[u8] = include_bytes!("../snapshots/smoke/data_image_roun
 
 /// Test for <https://github.com/linebender/vello/issues/972>
 #[test]
-// Note that this isn't exactly 0.3, it's a number starting with "0.3"
-#[should_panic(expected = "Expected mean to be less than 0.001, got 0.3")]
-fn test_data_image_roundtrip() {
+#[ignore]
```

Comment:
I'd like to see an ignore reason here. Something like "CI runs these tests on a CPU, leading to them having unrealistic precision".